modules/test/playwright/pages/document-library-web/documentLibrary.page.ts

```diff
@@ -37,9 +49,33 @@ export class DocumentLibraryPage {
 		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
 	}
 
+	async new() {
+		await this.page.getByRole('button', {name: 'New'}).click();
```

Comment:
I haven't a strong opinion here, because `New` and `Edit` buttons look important is fine to declare them, but if we are using var onetime inline it is always a good option for me.

Reply:
I see your point guys. At least in frontend, we were following the rule that @boton mentions, if it is only used once, there is no need to declare a variable. Here I would prefer to leave it inline.